src/package/packageVersionCreate.ts

```diff
@@ -464,19 +464,17 @@ export class PackageVersionCreate {
     // metadata exclusions. If necessary, read the existing package.xml and then re-write it.
     const currentPackageXml = await fs.promises.readFile(path.join(packageVersMetadataFolder, 'package.xml'), 'utf8');
     // convert to json
-    const packageJson = (await xml2js.parseStringPromise(currentPackageXml)) as {
-      Package: { types: Array<{ name: string[]; members: string[] }>; version: string };
-    };
-    if (!packageJson?.Package) {
+    const packageXmlAsJson = (await xml2js.parseStringPromise(currentPackageXml)) as PackageXml;
```

Comment:
not directly related, but we can remove the `clientSideInfo` param L459

Reply:
done. all destructured because 11 params is hard to manage